fathom/src/core.rs

```diff
@@ -403,14 +403,23 @@ def_prims! {
     PosAddU64 => "pos_add_u64",
 }
 
-/// Constants
+/// Formatting style for integers
 #[derive(Debug, Copy, Clone, PartialEq, PartialOrd)]
+pub enum IntStyle {
```

Comment:
Not sure if `UIntStyle` is a better name, given it is only used for unsigned integers. Not a big issue though - we can always defer this until we extend support to Signed integers as well.